packages/middleware-retry/src/isStreamingPayload/isStreamingPayload.browser.ts

```diff
@@ -0,0 +1,8 @@
+import type { HttpRequest } from "@smithy/protocol-http";
+
+/**
+ * @internal
+ */
+export const isStreamingPayload = (request: HttpRequest): boolean => {
+  return request?.body instanceof ReadableStream;
+};
```

Comment:
nit: brackets can be removed
```suggestion
export const isStreamingPayload = (request: HttpRequest): boolean => request?.body instanceof ReadableStream;
```